en/firefox/features/index.ftl

```diff
@@ -0,0 +1,114 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/features/
+
+
+# HTML page title
+features-index-protect-your-privacy-and-browse = Protect your privacy and browse faster with { -brand-name-firefox } features
+
+# HTML page description
+features-index-youre-in-control-with-firefoxs = You’re in control with { -brand-name-firefox }’s easy-to-use features that protect your privacy and browsing speeds.
+
+# Hero title
+features-index-firefox-features = { -brand-name-firefox } features
+
+# Hero description
+# Obsolete string
+features-index-explore-the-features-of-the = Explore the features of the all new { -brand-name-firefox } browser
+
+# Updated string
+features-index-firefox-is-fast = { -brand-name-firefox } is the fast, lightweight, privacy-focused browser that works across all your devices.
+
+# Obsolete string, replaced in firefox/features/shared.ftl
+features-index-your-favorite-add-ons-and = Your favorite add-ons and extensions
+features-index-add-powerful-functions = Add powerful functions, useful features and even a little fun to your Firefox browser.
+features-index-see-all = See all extensions
+
+# Obsolete string, replaced in firefox/features/shared.ftl
+features-index-customize-your-browser = Customize your browser
+features-index-give-your-browser = Give your browser the look you want  with thousands of different themes.
+
+# Obsolete string, replaced in firefox/features/shared.ftl
+features-index-sync-between-devices = Sync between devices
+features-index-important-stuff = Make sure all your important stuff — internet searches, passwords, open tabs — appears where you need it on every device.
+features-index-get-an-account = Get a { -brand-name-firefox-account }
+
+# Obsolete string
+features-index-tabs-that-travel = Tabs that travel
```

Comment:
# Obsolete string, replaced in firefox/features/shared.ftl `More powerful private browsing`